WalletWasabi.Fluent/Styles/Themes/BaseDark.axaml

```diff
@@ -28,26 +41,28 @@
 
     <SolidColorBrush x:Key="PrivacySuggestionControlSelectedBackground">#353535</SolidColorBrush>
 
-    <SolidColorBrush x:Key="TileRegionColor">#FF2B2B2B</SolidColorBrush>
-    <Color x:Key="RegionColor">#FF212121</Color>
-    <Color x:Key="HeaderRegionColor">#FF242424</Color>
+    <SolidColorBrush x:Key="TileRegionColor">#152032</SolidColorBrush>
+    <Color x:Key="RegionColor">#010D20</Color>
+    <Color x:Key="HeaderRegionColor">#010D20</Color>
     <BoxShadows x:Key="BoxShadow">0 1 4 #121212</BoxShadows>
     <BoxShadows x:Key="OptionButtonShadow">0 2 2 #121212</BoxShadows>
 
     <RadialGradientBrush x:Key="VersionHighlightRadialBrush" Radius="0.5" GradientOrigin="50%,15%" Center="50%,50%">
       <RadialGradientBrush.GradientStops>
-        <GradientStop Color="#F9FF5C" Offset="0"/>
-        <GradientStop Color="#86DF18" Offset="1"/>
+        <GradientStop Color="#F9FF5C" Offset="0" />
+        <GradientStop Color="#86DF18" Offset="1" />
       </RadialGradientBrush.GradientStops>
     </RadialGradientBrush>
 
     <SolidColorBrush x:Key="DataGridRowSelectedBackgroundColor" Color="#1C1C1C" />
-    <StaticResource x:Key="DataGridRowSelectedUnfocusedBackgroundColor" ResourceKey="DataGridRowSelectedBackgroundColor" />
+    <StaticResource x:Key="DataGridRowSelectedUnfocusedBackgroundColor"
+                    ResourceKey="DataGridRowSelectedBackgroundColor" />
 
     <StaticResource x:Key="NotificationCardBackgroundBrush" ResourceKey="TileRegionColor" />
   </Styles.Resources>
-  <StyleInclude Source="avares://Avalonia.Themes.Fluent/FluentDark.xaml"/>
-  <StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Fluent.xaml" /> <!-- Needs to be here because of dependency reason -->
+  <StyleInclude Source="avares://Avalonia.Themes.Fluent/FluentDark.xaml" />
+  <StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Fluent.xaml" />
+  <!-- Needs to be here because of dependency reason -->
```

Comment:
Can you put this back next to the DataGrid include line? It is misleading now.

Reply:
Rider hates inline comments in XAML so it always autocorrects that haha :) will just place it in its own line together with the datagrid